### DISCUSSION TITLE
DietPi-Software | Whoogle

### DISCUSSION THREAD

**Discussion by caio1007** [MichaIng/DietPi#6215] [Software Request]
**Creating a software request**

_Formal software information_

- **Software title | Whoogle**

- **Short description |** Get Google search results, but without any ads, javascript, AMP links, cookies, or IP address tracking. Easily deployable in one click as a Docker app, and customizable with a single config file. Quick and simple to implement as a primary search engine replacement on both desktop and mobile.

- **GitHub URL (if available) |** https://github.com/benbusby/whoogle-search

- **(Official) install documentation |** https://github.com/benbusby/whoogle-search

**Comment by Joulinar** [comment]
Please vote for this software implementation via ⬆️ button at first post.
